etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/localfile/FilePartnerMetadataStorage.java

```diff
@@ -23,14 +22,13 @@ public class FilePartnerMetadataStorage implements PartnerMetadataStorage {
     @Inject Formatter formatter;
     @Inject Logger logger;
 
-    private static final Path metadataTempDirectory;
+    private static final Path metadataDirectory;
 
     static {
         try {
-            FileAttribute<?> onlyOwnerAttrs =
-                    PosixFilePermissions.asFileAttribute(
-                            PosixFilePermissions.fromString("rwx------"));
-            metadataTempDirectory = Files.createTempDirectory("metadata", onlyOwnerAttrs);
+            Path projectRoot = Paths.get(System.getProperty("user.dir"));
```

Comment:
Is this a reliable way to get the root path?

Reply:
After discussing with the team, decided to change this implementation to create a folder in the system temp path to put the files in